hera_cal/lstbin.py

```diff
@@ -422,23 +424,19 @@ def lst_align_arg_parser():
     return a
 
 
-def lst_align_files(data_files, file_ext=".L.{:7.5f}", dlst=None,
+def lst_align_files(data_files, file_ext=".L.{:7.5f}", dlst=None, filetype='uvh5',
```

Comment:
It seems like this function and its associated arg_parser and script are never used. Can we just remove them?

Reply:
sure